packages/tools/client-debugger/client-debugger-view/src/RootView.tsx

```diff
@@ -27,7 +28,9 @@ export interface RootViewProps {
 export function RootView(props: RootViewProps): React.ReactElement {
 	return (
 		<MessageRelayContext.Provider value={props.messageRelay}>
-			<FluidClientDebuggers />
+			<FluentProvider theme={teamsLightTheme}>
```

Comment:
Once we apply the theme globally using `getFluentUIThemeToUse()` it will also switch light/dark based on browser setting.